mod/contacts.php

```diff
@@ -855,6 +855,10 @@ function _contact_detail_for_template($rr){
 		$sparkle = '';
 	}
 
+	//test if contact is a forum page
+	if (isset($rr['forum']) OR isset($rr['prv']))
+				$community = ($rr['forum'] OR $rr['prv']);
```

Comment:
Is the variable $community defined before? It should be set to false by default - although the case should not happen that the two array values are undefined.